pkg/metrics/controller_metrics.go

```diff
@@ -52,6 +53,7 @@ var (
 	fasLimitedStats           = stats.Int64("fas/limited", "The fleet autoscaler is capped (0 indicates false, 1 indicates true)", "1")
 	gameServerCountStats      = stats.Int64("gameservers/count", "The count of gameservers", "1")
 	gameServerTotalStats      = stats.Int64("gameservers/total", "The total of gameservers", "1")
+	gameServerPlayersInGame   = stats.Int64("gameservers/players_in_game", "The total number of players connected to gameservers", "1")
```

Comment:
Per https://prometheus.io/docs/practices/naming/ I've suggest making this `gameserver_player_connected_total` for the total number of players.

Since you are in here, should we also add `gameserver_player_capacity_total` such that you can also track how much room there is over time?